internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -385,7 +389,7 @@ Resources:
               - !Sub arn:aws:s3:::${Bucket}
               - !Sub arn:aws:s3:::${Bucket}/*
 
-{{- if .StaticSiteAlias}}
+{{- if and .StaticSiteAlias (not .StaticSiteCert)}}
```

Comment:
wait don't we still need to add A records pointing to CF? my understanding is:
```
if alias: need CustomDomain and CertificateValidator
if cert (implies alias set as well): need CustomDomain
if neither: don't need either 
```

Reply:
i see, that does make sense.... i just wonder if people will be confused when it doesn't work until they put in the r53 entry manually 🤔

Reply:
Gotcha. Maybe it makes more sense to add this to the doc.